test/GetPrefilterExpressionFromSparqlExpressionTest.cpp

```diff
@@ -612,3 +618,17 @@ TEST(GetPrefilterExpressionFromSparqlExpression,
           "The vector must contain the <PrefilterExpression, Variable> "
           "pairs in sorted order w.r.t. Variable value."));
 }
+
+//______________________________________________________________________________
+// Test helper `getStringViewFromLiteralExpression` from LiteralExpression.h
+TEST(GetPrefilterExpressionFromSparqlExpression,
+     getNormalizedStringViewFromStringLiteralExpression) {
+  using namespace sparqlExpression;
+  ASSERT_TRUE(
+      sparqlExpression::detail::getStringViewFromLiteralExpression(
+          std::make_unique<StringLiteralExpression>(L("\"hello\"")).get())
+          .has_value());
```

Comment:
The important part is missing: Does this include the `"` or not, so what is the `value()` of the result.